master/requirements.yaml

```diff
@@ -0,0 +1,61 @@
+dependencies:
+  - name: andi
+    version: 1.1.0
```

Comment:
Should these chart versions be overridable? Seems like users might want to pin what chart version they use.

Reply:
Our thought was to just get a version-set that worked on local. I'm not opposed to making it overridable though.

Reply:
I created an issue for this
smartcitiesdata/charts#88